### PR TITLE
Check seekable().length before calling seekable().end(0)

### PR DESCRIPTION
**Pull request by incompl**

### CODE REVIEW

File: src/videojs.ads.js [hunk 1]
```diff
@@ -68,7 +68,12 @@ var
 
     if (videojs.browser.IS_IOS && isLive(player)) {
       // Record how far behind live we are
-      currentTime = player.currentTime() - player.seekable().end(0);
+      try {
+        currentTime = player.currentTime() - player.seekable().end(0);
```
Comment (gkatsev) [inline src/videojs.ads.js]: Why not
```js
if (player.seekable().length > 0) {...}
```
Error out on `seekable()` in generaly?

Comment (incompl) [inline src/videojs.ads.js]: I changed it to that

**Comment by dmlap** [comment]
LGTM

**Comment by misteroneill** [comment]
LGTM

**Pull request merged**